### ISSUE TITLE
Support of docker-ce yum repository for CentOS 8 Stream after CentOS 8 deprecation

### ISSUE THREAD

**Issue by cvstealth** [docker/for-linux#1222]
* [x] This is a bug report
* [ ] This is a feature request
* [x] I searched existing issues before opening this one

With the deprecation of CentOS 8 slated for Dec 2021, will the docker-ce yum repositories still be updated with current/newer versions of docker for use on CentOS 8 Stream?

**Comment by linghengqian** [comment]
This issue has been resolved as it is Fedora in nature and should be closed.

**Comment by thaJeztah** [comment]
Yes, packages for CentOS stream 8 and 9 are available on download.docker.com; 
https://download.docker.com/linux/centos/8/
https://download.docker.com/linux/centos/9/
